python/llm/src/ipex_llm/transformers/models/chatglm4v.py

```diff
@@ -99,10 +97,11 @@ def chatglm4v_model_forward(
     use_cache = use_cache if use_cache is not None else self.config.use_cache
     return_dict = return_dict if return_dict is not None else self.config.use_return_dict
 
-    batch_size, seq_length = input_ids.shape
-
     if inputs_embeds is None:
+        batch_size, seq_length = input_ids.shape
         inputs_embeds = self.embedding(input_ids)
+    else:
+        batch_size, seq_length, _ = inputs_embeds.shape
```

Comment:
`input_ids` is needed for `self.get_masks()`. Maybe add an empty tensor here (similar to chatglm2/4)

Reply:
> `input_ids` is needed for `self.get_masks()`. Maybe add an empty tensor here (similar to chatglm2/4)

Have updated :)